src/runtimes/node/bundlers/esbuild/src_files.ts

```diff
@@ -15,11 +15,12 @@ export const getSrcFiles: GetSrcFilesFunction = async ({ config, mainFile, plugi
     basedir: srcDir,
     pluginsModulesPath,
   })
-  const includedPaths = filterExcludedPaths([...dependencyPaths, ...includedFilePaths], excludedPaths)
+  const srcFiles = filterExcludedPaths(dependencyPaths, excludedPaths)
+  const includedPaths = filterExcludedPaths(includedFilePaths, excludedPaths)
 
   return {
-    srcFiles: [...includedPaths, mainFile],
-    includedFiles: filterExcludedPaths(includedFilePaths, excludedPaths),
+    srcFiles: [...srcFiles, ...includedPaths, mainFile],
+    includedFiles: includedPaths,
```

Comment:
I just made some minor adjustments to evrey bundler, so that we do not iterate over includedFilePaths twice.

Reply:
Nice! That also makes things a bit clearer, IMO.